### ISSUE TITLE
Box around current room in LeftPanel doesn't stand out

### ISSUE THREAD

**Issue by eternaleye** [vector-im/riot-web#8501]
The LeftPanel background is #f2f5f8, against which the selected room's #fff RoomTile background is basically invisible

**Comment by lampholder** [comment]
And the WCAG AA Contrast Checker result is... FAIL

![image](https://user-images.githubusercontent.com/1922197/53011717-f1bcb580-3438-11e9-97f0-6676d703b5fd.png)
(https://webaim.org/resources/contrastchecker/)

**Comment by inducer** [comment]
This has also made Ctrl+K/<Search term>/Tab/Tab/Tab a lot less useful for me, because I can't tell which room I'll go to when I hit Enter. I haven't dug into the code, so I don't know for sure, but this seems like it should be an easy CSS fix. Help?

**Comment by lampholder** [comment]
The CSS change is trivial - the tricky bit is getting this to the top of our designer's todo to get some attention. I'll see what he thinks about proposing a quick fix...

**Comment by nadonomy** [comment]
Having looked at this, although the CSS implementation is trivial, there's unfortunately no 'cheap fix' that doesn't involve dramatically changing the look & feel of the app altogether, which is subject to approval by way of many discussions and pleasing multiple stakeholders.

That said, the utility of the highlight is to tell users which room is selected, which they do know otherwise (it's being rendered, after all). As this specific issue is aesthetic/stylistic I'm downgrading this to p2.

**Comment by inducer** [comment]
It's really functional, too: Suppose I use the keyboard shortcut (Ctrl+K) for filtering the list of rooms, type in a search term, and then tab through the filtered list: The highlight is my only indication which room I might select when I hit Enter, and it doesn't work very well in this role.

**Comment by t3chguy** [comment]
The accessibility of this has now been improved, the default outline has been restored when using keyboard navigation: 
![image](https://user-images.githubusercontent.com/2403652/66109000-a74d0b00-e5bb-11e9-9156-319253dbb5f2.png)

**Comment by t3chguy** [comment]
I am not sure if this issue describes any additional issues, feel free to close if this is sufficient

**Comment by inducer** [comment]
That should work, feel free to close. I'll look at it once I have that deployed, and if something is still off, I'll comment/reopen here or make a fresh issue.
